solvebio/cli/data.py

```diff
@@ -633,26 +640,51 @@ def _download(
             )
         )
 
+    if not follow_shortcuts:
```

Comment:
Should we inform users e.g. "5 shortcuts were found but not downloaded, use `--follow-shortcuts` flag to download."?

Reply:
E.g. when I tried to download from a vault with only a shortcuts, I get the following message:

```
❯ solvebio download --recursive "admin-qb-int-dev:nish-2-vault:/" .
Found 0 files to download.
No files found on path.
```

Since we have the information of how many shortcuts there are, we should at least let the user know.

Reply:
You're right. I'll add this to the base 'recursive_download' branch and then merge into this one, once finalized there.